### DISCUSSION TITLE
Testing if a target is a clone

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#4714] [Help] [answered]
I'm writing an addon that has a circumstance where being able to differentiate between a sprite and it's clone would be beneficial. If anyone knows a way to do this, that'd be amazing. I'm pretty new to developing addons and don't quite know all the functions and properties yet.

**Comment by GarboMuffin** [comment, marked as answer]
`!target.isOriginal`

if that doesn't seem to make sense then share more of your code

**Reply by ghost** [reply]
Great, thanks! I love how active the forum is. It makes things a lot easier

**Reply by ghost** [reply]
The variable manager that you worked on is actually the base of my addon, the sprite layering manager.
